src/App.css

```diff
@@ -1,11 +1,11 @@
 body {
-  display: flex;
-  justify-content: center;
-  padding-top: 200px;
+  background-color: #ffdee9;
+  background-image: linear-gradient(90deg, #ffdee9 0%, #b5fffc 100%);
 }
 
 .container {
   width: 420px;
+  padding-left: 50px;
 }
 
 .row-div {
```

Comment:
- Hey, please kindly follow the placement of the calculator as shown in the 
[wireframe](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/images/math_magicians_calculator.png) and also add the missing text at the left "Let's do some maths" to meet the wireframe as much as you can. below is your design:


![Screenshot from 2022-11-10 22-12-29](https://user-images.githubusercontent.com/59397159/201207878-4ac6c191-b7a4-45ee-ac57-2918a22efad2.png)